deploy/olm-catalog/opendatahub/1.4.1/manifests/opendatahub-operator.v1.4.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,289 @@
+                    "path": "model-mesh"
+                  }
+                },
+                "name": "model-mesh"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "metadata-store-mariadb",
+                    "ds-pipeline-ui",
+                    "object-store-minio",
+                    "default-configs"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "data-science-pipelines"
+                  }
+                },
+                "name": "ds-pipelines"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "grafana/cluster"
+                  }
+                },
+                "name": "grafana-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "grafana/grafana"
+                  }
+                },
+                "name": "grafana-instance"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/cluster"
+                  }
+                },
+                "name": "prometheus-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "prometheus/operator"
+                  }
+                },
+                "name": "prometheus-operator"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "additional"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "jupyterhub/notebook-images"
+                  }
+                },
+                "name": "notebook-images"
+              }
+            ],
+            "repos": [
+              {
+                "name": "manifests",
+                "uri": "https://github.com/opendatahub-io/odh-manifests/tarball/v1.4"
+              }
+            ]
+          }
+        }
+      ]
+    capabilities: Deep Insights
+    categories: AI/Machine Learning, Big Data
+    description: "Open Data Hub Operator for deployment and management of Open Data Hub and Kubeflow on OpenShift"
+    support: Open Data Hub
+    repository: https://github.com/opendatahub-io/opendatahub-operator
+    createdAt: '2022-10-18T00:00:00Z'
+    containerImage: quay.io/opendatahub/opendatahub-operator:v1.4.1
+    certified: 'False'
+  name: opendatahub-operator.v1.4.1
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
```

Comment:
Is there a reason we did not add `OdhQuickstart` crd to this list?

After testing in a 4.12 cluster, I see that all CRDs now have same labels as Kfdef except `OdhQuickstart`

```
oc get crds -l operators.coreos.com/opendatahub-operator.openshift-operators
NAME                                       CREATED AT
kfdefs.kfdef.apps.kubeflow.org             2022-12-06T04:38:19Z
odhapplications.dashboard.opendatahub.io   2022-12-06T04:38:19Z
odhdashboardconfigs.opendatahub.io         2022-12-06T04:38:20Z
odhdocuments.dashboard.opendatahub.io      2022-12-06T04:38:21Z
```